### PR TITLE
Maxwell histogram

### PR DESCRIPTION
**Pull request by lilyclements**

### CODE REVIEW

File: instat/dlgHistogram.vb [hunk 1]
```diff
@@ -169,7 +169,7 @@ Public Class dlgHistogram
             clsRgeomPlotFunction.SetRCommand("geom_freqpoly")
             ucrSaveHist.SetPrefix("FrequencyPolygon")
             ucrFactorReceiver.ChangeParameterName("colour")
-            cmdHistogramOptions.Text = "Freequency Polygon Options"
+            cmdHistogramOptions.Text = "Frequency Polygon Options"
```
Comment (lilyclements) [inline instat/dlgHistogram.vb]: Changed name from Freequency to Frequency

File: instat/dlgHistogram.vb [hunk 2]
```diff
@@ -82,17 +81,16 @@ Public Class dlgHistogram
...
-        ucrHistogramSelector.Reset()
-        ucrSaveHist.Reset()
-
-        sdgPlots.Reset()
-        TempOptionsDisabledInMultipleVariablesCase()
-
         clsBaseOperator = New ROperator
         clsRggplotFunction = New RFunction
         clsRgeomPlotFunction = New RFunction
         clsRaesFunction = New RFunction
```
Comment (lilyclements) [inline instat/dlgHistogram.vb:87]: Moved the Operators/Functions to the top of the sub

File: instat/dlgHistogram.vb [hunk 3]
```diff
@@ -113,7 +111,7 @@ Public Class dlgHistogram
         TestOkEnabled()
     End Sub
 
-    Private Sub SetRCodeForControls(bReset As Boolean)
+    Public Sub SetRCodeForControls(bReset As Boolean)
```
Comment (lilyclements) [inline instat/dlgHistogram.vb]: Public sub for SetRCode?

**Review by lilyclements — COMMENTED** [review]
@maxwellfundi This is ready to review

**Comment by maxwellfundi** [comment]
@lilyclements too many files changed here, I will just manually have to take those changes and have them manually.

**Comment by lilyclements** [comment]
When i merge things in, they usually have loads of file changes because my branch is more updated. It doesn't cause a conflict for others when it goes in, and it hasn't caused conflicts foryou in the past.
I think you can merge this without any issues with conflicts. @dannyparsons is this a correct statement?!

**Comment by dannyparsons** [comment]
As long as Lily's branch is more up to date than Maxwell's the merge should be fine. Also remember you can revert a commit so if this causes problems you can always reverse back to before you merged this in. So I think there's no harm in merging this.

**Pull request merged**